src/Microsoft.DotNet.Tools.Compiler.Native/IntermediateCompilation/Linux/LinuxCppCompileStep.cs

```diff
@@ -87,10 +87,11 @@ private void InitializeArgs(NativeCompileSettings config)
             }
 
             // AppDep Libs
+            var baseAppDeplibPath = Path.Combine(config.AppDepSDKPath, "CPPSdk/ubuntu.14.04/x64");
```

Comment:
This is not for managed DLLs but native binaries (i.e. libSystem.Native.a) which are currently in AppDep package root. The headers are common, so I dont believe we have a need for them to be in platform/arch specific folder.